Gopkg.toml

```diff
@@ -38,16 +38,12 @@
   name = "github.com/fatih/color"
   version = "1.1.0"
 
-[[constraint]]
-  name = "github.com/giantswarm/api-schema"
-  version = "0.7.1"
-
 [[constraint]]
   name = "github.com/giantswarm/columnize"
   version = "=2.0.2"
 
 [[constraint]]
-  branch = "master"
+  branch = "v4-login"
   name = "github.com/giantswarm/gsclientgen"
```

Comment:
should it be changed back to master after test?

Reply:
Yes, as it says in the main comment.